collab-folder/src/core/relation.rs

```diff
@@ -28,6 +28,60 @@ impl ViewRelations {
     })
   }
 
+  /// Disconnect relation between parent_id and view_id
+  pub fn disconnect_child_with_txn(
```

Comment:
I think these function name would be more precise.

dissociateParentChild()
detachParentChild()

associateParentChild()
attachParentChild()